index_test.go

```diff
@@ -3254,7 +3254,7 @@ func TestLineAnd(t *testing.T) {
 
 	t.Run("ChunkMatches", func(t *testing.T) {
 		res := searchForTest(t, b, &q, chunkOpts)
-		wantRegexpCount := 1
+		wantRegexpCount := 2 // TODO: justify this change
```

Comment:
justification?

Reply:
Took a while to find it (first time I've ever had two debuggers going in parallel), but I had broken a perf optimization. I missed a spot where I needed to change `<= to `<`. Glad I left the TODO, and thanks for reminding me!